repository_rules/framework_builder.bzl

```diff
@@ -15,13 +15,16 @@ def _create_buildfile_content(root, frameworks_subpath):
     binaries_path = root
     for element in frameworks_subpath.split("/"):
         binaries_path = binaries_path.get_child(element)
+        nested_framework_binaries_path = binaries_path.get_child("Frameworks")
+        if nested_framework_binaries_path.exists:
+            binaries_path = nested_framework_binaries_path.readdir()[0]
```

Comment:
Hey :) I dont think this is correct, this function `_create_buildfile_content` creates the buildfile content for an **already found** framework: it should not know anything related with the cocoapods folder structure.

I think what you need instead is to add the following after line 170:
```
    for vendoredFrameworkPath in path.get_child("Frameworks").readdir():
        filegroups = _create_buildfile_content(wd, prebuild_subpath + "/" + path.basename + "/Frameworks/" + vendoredFrameworkPath.basename())
        buildfile_content = buildfile_content + filegroups
```

Keep in mind I did not run the code above, so it may be no 100% working, but I think the idea should be something like that